ouroboros-consensus/src/ouroboros-consensus/Ouroboros/Consensus/Ledger/Tables.hs

```diff
@@ -8,12 +8,26 @@
 {-# LANGUAGE TypeOperators            #-}
 {-# LANGUAGE UndecidableInstances     #-}
 
--- | See @'LedgerTables'@
+-- | This module defines the 'LedgerTables', a portion of the Ledger notion of a
+-- /ledger state/ (not to confuse with our
+-- 'Ouroboros.Consensus.Ledger.Basics.LedgerState') that together with it,
+-- conforms a complete Ledger /ledger state/.
+--
+-- In particular, 'LedgerTables' contain maps from keys to values. For now,
```

Comment:
Do they? Sometimes ledger tables are instantiated to sets or diffs, aren't they? If so we might want to give the reader the false impression that tables are always k-v maps.